idx/cassandra/cassandra.go

```diff
@@ -680,6 +681,27 @@ func (c *CasIdx) Delete(orgId uint32, pattern string) ([]idx.Archive, error) {
 	return defs, err
 }
 
+func (c *CasIdx) DeleteTagged(orgId uint32, query tagquery.Query) ([]idx.Archive, error) {
+	pre := time.Now()
+	defs, err := c.MemoryIndex.DeleteTagged(orgId, query)
+	if err != nil {
+		return nil, err
+	}
+
+	if c.Config.updateCassIdx {
+		for _, def := range defs {
+			delErr := c.deleteDef(def.Id, def.Partition)
+			if delErr != nil {
+				log.Errorf("cassandra-idx: %s", delErr.Error())
+				err = delErr
+			}
+		}
+	}
```

Comment:
Minor, but it might make sense to pull this into a method since it's used both here and in `Delete` (same for the bigtable version)

Reply:
good point, how is that 7c82fbaf6d8340b184e268e5dcc294d7de061535

Reply:
LGTM